actions/payment.js

```diff
@@ -48,9 +50,15 @@ exports.list = class List extends Action {
           ]
         } : {})
       },
-      order: [['paymentDate', 'DESC']],
       ...(limit !== -1 && !memberId ? { offset, limit } : {})
     }
+    if (order != null) {
```

Comment:
same as members ordering

```suggestion
    if (order) {
```